packages/dart/npt_flutter/lib/features/policy/cubit/policy_state.dart

```diff
@@ -0,0 +1,137 @@
+part of 'policy_cubit.dart';
+
+/// Comprehensive enum representing different view modes in the Policy Manager
+enum PolicyViewMode {
+  /// Browsing the list of roles with no selection
+  rolesBrowsing,
+  
+  /// Viewing details of an existing role (read-only)
+  roleViewing,
+  
+  /// Editing an existing role
+  roleEditing,
+  
+  /// Creating a new role
+  roleCreating,
+  
+  /// Viewing policy logs (completely separate from roles)
+  logsViewing,
+}
```

Comment:
This should be represented as a sealed covariant of ProfileLoaded. See https://github.com/atsign-foundation/noports/blob/trunk/packages/dart/npt_flutter/lib/features/profile/bloc/profile_state.dart